packages/vue-dot/src/patterns/DatePicker/DatePicker.vue

```diff
@@ -6,13 +6,14 @@
 		v-bind="options.menu"
 	>
 		<!-- eslint-disable-next-line vue/no-unused-vars -->
-		<template #activator="{ on }">
+		<template #activator="{}">
 			<!-- TextField to enter date by hand -->
 			<VTextField
 				ref="input"
 				v-model="dateFormatted"
 				v-mask="maskValue"
 				v-bind="options.textField"
+				:error-messages="options.errorMessages"
```

Comment:
```suggestion
				:error-messages="options.textField.errorMessages"
```
Je ne suis pas d'accord ici !
La propriété calculée `options` vient de la propriété `vuetify-options`, sa structure est prédéfinie comme ceci : chaque composant Vuetify du composant actuel a une propriété dédiée qui sert à configurer les options de Vuetify

Donc si on a un `VTextField`, dans mes `vuetify-options`, je vais avoir une propriété `textField` dans laquel je vais pouvoir configurer (par exemple) les messages d'erreur.
Il ne faut pas changer cette logique ! Sinon on perd le sens de ce concept et ça sera impossible à documenter.